### ISSUE TITLE
Tests for fully connected tensorflow models

### ISSUE THREAD

**Issue by rbharath** [deepchem/deepchem#150]
The fully connected tensorflow models train successfully, with the train loss tending to zero. However, these models seem to learn the zero classifier. Given the complexity of the `add_training_cost` methods for the `TensorflowClassifier` class, we need to add some simple artificial datasets which we can use to test that the models have correct loss functions.

**Comment by rbharath** [comment]
Related to #164

**Comment by rbharath** [comment]
These tests have been added in follow-on PRs.
